pages/1.10/security/ent/tls-ssl/ca-custom/index.md

```diff
@@ -8,26 +8,306 @@ excerpt:
+The parameter must be left undefined if `ca_certificate_path` points to a root CA certificate.
+Required if `ca_certificate_path` is specified and if the custom CA certificate is an *intermediate CA certificate*. 
+
+For a non-root intermediate CA, this needs to point to a file containing all CA certificates comprising the complete sequence starting precisely with the CA certificate that was used to sign the custom CA certificate and ending with a root CA certificate (where issuer and subject are equivalent), yielding a gapless certification path. The order is significant and the list must contain at least one certificate.
+
+
+# Installing DC/OS Enterprise with a custom CA certificate
+## Starting point
+Based on the requirements described above, this is the starting point for the installation:
+
+- The installation of DC/OS Enterprise via the Advanced Installer has been prepared according to the corresponding [documentation](/1.10/installing/ent/custom/advanced/). (up to the section **Install DC/OS**)
+
+- On the bootstrap node, the files carrying custom CA certificate, the associated private key and, optionally, the CA certificate chain have been placed into the `$DCOS_INSTALL_DIR/genconf/` directory. (see the [section](#manually-placing-the-custom-ca-certificate-the-associated-private-key-and-the-certificate-chain-onto-the-bootstrap-node) above for more detailed description)
+
+- The custom CA private key has been securely placed on all DC/OS master nodes and is sufficiently protected (refer to [this section](#manually-placing-the-private-key-associated-with-the-custom-ca-certificate-onto-the-master-nodes) for more details). Example (command issued on one of the DC/OS master nodes):
+
+```bash
+stat /var/lib/dcos/pki/tls/CA/private/custom_ca.key
+```
+```bash
+File: ‘/var/lib/dcos/pki/tls/CA/private/custom_ca.key’
+Size: 9             Blocks: 8          IO Block: 4096   regular file
+Device: ca01h/51713d    Inode: 100671105   Links: 1
+Access: (0600/-rw-------)  Uid: (    0/    root)   Gid: (    0/    root)
+Context: unconfined_u:object_r:var_lib_t:s0
+Access: 2017-12-27 12:35:58.643278377 +0000
+Modify: 2017-12-27 12:35:58.643278377 +0000
+Change: 2017-12-27 12:36:13.019162417 +0000
+Birth: -
+```
+
+- The configuration parameters `ca_certificate_path`, `ca_certificate_key_path` and `ca_certificate_chain_path` are specified in the DC/OS configuration file `$DCOS_INSTALL_DIR/genconf/config.yaml` and point to the relevant locations in the file system. Example (commands issued on the bootstrap node):
+
+```bash
+cd $DCOS_INSTALL_DIR
+cat genconf/config.yaml
+```
+```bash
+[...]
+ca_certificate_path: genconf/dcos-ca-certificate.crt
+ca_certificate_key_path: genconf/dcos-ca-certificate-key.key
+ca_certificate_chain_path: genconf/dcos-ca-certificate-chain.crt
+[...]
+```
+Note that `ca_certificate_chain_path` must not be present when setting up DC/OS Enterprise with a root certificate as the custom CA certificate.
```

Comment:
Again, to avoid ambiguity, "...with a self-signed root CA certificate..."

Reply:
The glossary actually mentions that the root CA certificate is one which is self-signed, so it should not be ambiguous. But in principle it does not hurt to mention it once again. @jgehrcke  asked in another comment to change "self-signed root CA certificate" to "root CA certificate", that's why I removed it through the doc entirely.

Reply:
OK, fine to leave it as is then.